lib/hibp/parsers/password.rb

```diff
@@ -21,15 +22,15 @@ class Password
       def parse_response(response)
         data = response.body
 
-        data.split(ROWS_SPLITTER).map(&method(:convert_to_password))
-      end
-
-      private
+        data.split(ROWS_SPLITTER).inject([]) do |array, row|
```

Comment:
Sorry, this got a little messy 😅 . Any passwords that have 0 occurrences are the fake records the padding option adds to the response, and returning those to the user seemed pointless since they'd have to filter them out anyway to get useful information. I didn't want to use `filter` and `select` to avoid iterating over the data twice and `filter_map` is only Ruby 2.7, so I didn't want to make that a dependency either. Hence, the inject method. If you have a solution you'd prefer happy to implement that instead 😄

Reply:
I am totally ok with this